crates/primitives/src/chain_evm.rs

```diff
@@ -29,4 +30,63 @@ impl EVMChain {
     pub fn all() -> Vec<EVMChain> {
         EVMChain::iter().collect::<Vec<_>>()
     }
+
+    pub fn min_priority_fee(&self) -> u64 {
+        match self {
+            EVMChain::Ethereum => 1000000000, // https://etherscan.io/gastracker
```

Comment:
```suggestion
            EVMChain::Ethereum => 1_000_000_000, // https://etherscan.io/gastracker
```

can you adjust for all values? easier to read